api/package.json

```diff
@@ -20,8 +20,8 @@
     "coverage": "nyc mocha",
     "lint": "eslint . --ignore-pattern 'node_modules' --ext .ts",
     "lint:fix": "eslint . --fix --ignore-pattern 'node_modules' --ext .ts",
-    "format": "prettier --write \"./src/**/*.{js,jsx,ts,tsx,json,css,scss}\"",
-    "check": "prettier --check \"./src/**/*.{js,jsx,ts,tsx,css,scss}\""
```

Comment:
None of the contents of these commands changed (in any of the 4 package.json files), I only changed the name.